modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/toolbarbuttons/plugin.js

```diff
@@ -21,6 +23,14 @@
 
 	CKEDITOR.plugins.add(pluginName, {
 		init(editor) {
+			const path = this.path;
+
+			const dependencies = [CKEDITOR.getUrl(path + 'LinkUtils.js')];
+
+			CKEDITOR.scriptLoader.load(dependencies, () => {
```

Comment:
I would like to conditionally add this JS here when the twitter button is being passed

Reply:
It doesn't seem right that this is always executed. Is there an `init()` in `addBalloonToolbarButton` like there is for `addRichCombo`?

Reply:
It requires adding a callback like [you have done with `onRender` in ui-select](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/uiselect/plugin.js#L148) and when examining about I don't have idea where to place this onInit. Could be [here](https://github.com/liferay/liferay-portal/blob/a2e49843a17f71c92c4709fe77f09d3e2e775cc6/modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/uibutton/plugin.js#L111)?